www/gatsby-config.js

```diff
@@ -306,7 +306,20 @@ module.exports = {
         ],
       },
     },
-    `gatsby-plugin-netlify`,
+    {
+      resolve: `gatsby-plugin-netlify`,
+      options: {
+        mergeSecurityHeaders: false,
```

Comment:
Setting this to true (or omitting this because default is true) should merge headers so not everything needs to be duplicated. If this is not the case that's a bug in gatsby-plugin-netlify :joy:.
```suggestion
        mergeSecurityHeaders: false,
```

Reply:
thats strange. im having a hard time getting build to run locally due to some mdx errors elsewhere, but i tested this in the browser and it seems to be running as expected, not sure what i was seeing before 🤔made the change
![image](https://user-images.githubusercontent.com/1045274/67282371-a19e6300-f49f-11e9-9384-73f8b0eb31cf.png)

local issues im running into. im not seeing this in the codebase temp remove it. post `yarn install` fwiw if that matters
![image](https://user-images.githubusercontent.com/1045274/67282472-df02f080-f49f-11e9-9c0c-9696863192c9.png)